### PR TITLE
Chapter 1 grammatical review

### PR DESCRIPTION
**Pull request by Aerek-Yasa**
I read through chapter 1. I fixed some minor grammatical mistakes and changed a couple sentences that felt unnatural. I'm not entirely sure about changing "versatility" to "expressiveness" on line 241 though.

~Aerek

### CODE REVIEW

File: book/01.md [hunk 1]
```diff
@@ -238,29 +238,29 @@ End of TN.
 
 The DB instruction is an abbreviation for "Data Byte". That is to write 1 byte directly to the file. I like writing assembly instructions in uppercase, but they can be written in lowercase like "`db`" without any problems.
 
-This instruction is the strongest ultimate weapon in the world of assembler, that is, any data (and any program, by the way) can be written with just that instruction. That's why there are no files that can't be created in assembler. You can create text files, image files, whatever you're told to do, you can do it in assembler. This versatility(?) is rare in other languages (e.g. C).
+This instruction is the ultimate strongest weapon in the world of assembler. That is, any data (and any program, by the way) can be written with just that instruction. That's why there are no files that can't be created in assembler. You can create text files, image files, whatever you're told to do, you can do it in assembler. This expressiveness is rare in other languages (e.g. C).
```
Comment (flysand7) [inline book/01.md]: >expressiveness

In the original book that part of the sentence says 

```
他の言語（たとえばC言語）ではこのような万能性（？）はめったにありません
```
Google translate interprets the word 万能性 as "versatile", and on the Japanese side the part "万能" means "omnipotence", and as far as I know "性" means "characteristic". So in this case author definitely tries to reach the thought that it's the fact that it literally can do anything and be anything. I'm not quite sure about the meaning of the (?) though, so my best treatment would be to leave it there. Anyway I don't think "expressiveness" quite captures the original meaning

File: book/01.md [hunk 2]
```diff
@@ -238,29 +238,29 @@ End of TN.
...
 
 The RESB instruction is an abbreviation of "Reserve Byte", with a meaning of something like "For now just leave 10 bytes there" you'd use it like RESB 10. And these 10 bytes are reserved (imagine something like a reserved seat). But in NASK it doesn't just reserve the bytes but to fill them with zeroes. So we're using this command to write a bunch of 0x00. Thanks to this instruction I don't have to write 180000 lines, which is a big help.
 
 ---
 
-TN: The implication is that in NASM it will reserve zeroes but, will hesitate to fill them with zeroes. While actually filling the file with zeroes it prints warning like this:
+TN: The implication is that in NASM it will also reserve zeroes but, it will hesitate. While actually filling the file with zeroes it prints a warning like this:
```
Comment (flysand7) [inline book/01.md]: Should be "reserve space", not "reserve zeroes", that was a thoughtless typo on my side. Reserving zeroes makes no sense to begin with :D

**Review by flysand7 — CHANGES_REQUESTED** [review]
Some parts require more big think

File: book/01.md (hunk 1 above)
Comment (Aerek-Yasa) [reply book/01.md]: Reverted to versatile for now. It does seem like the better fit. Also added back in the question mark.

File: book/01.md (hunk 2 above)
Comment (Aerek-Yasa) [reply book/01.md]: You're right, that does make more sense. Changed it to reserve space. Should be resolved now!

**Pull request merged**